pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
@turtleli - what's your opinion ? Is there some sort of bug on ``CorePlugins`` which indicates that the GS plugin is initialized even after a shutdown ? If yes, this needs to be fixed instead of omitting `` || CorePlugins.AreAnyInitialized()`` on the ``Enable()`` function. That's my opinion on the matter.